cpu/sam0_common/periph/rtt.c

```diff
@@ -36,18 +36,28 @@
 #define RTC_MODE0_CTRLA_COUNTSYNC       (0x1ul << RTC_MODE0_CTRLA_COUNTSYNC_Pos)
 #endif
 
+#ifdef REG_RTC_MODE0_CTRLA
+#define RTC_MODE0_PRESCALER       \
+    (__builtin_ctz(RTT_CLOCK_FREQUENCY / RTT_FREQUENCY) << \
+    RTC_MODE0_CTRLA_PRESCALER_Pos)
+#else
+#define RTC_MODE0_PRESCALER       \
+    (__builtin_ctz(RTT_CLOCK_FREQUENCY / RTT_FREQUENCY) << \
+    RTC_MODE0_CTRL_PRESCALER_Pos)
```

Comment:
Tested on iotlab for `samr30-xpro` and figure out there is a difference  between `samd21` and `saml21` I'll add a patch.

Reply:
![image](https://user-images.githubusercontent.com/23060007/84991759-419faf80-b147-11ea-9778-5bcb558dabd1.png)

![image](https://user-images.githubusercontent.com/23060007/84991779-482e2700-b147-11ea-884f-a57d636ba564.png)

Reply:
Sam applies for the other sub-families.

Reply:
c11dcf7 addresses this